src/App.tsx

```diff
@@ -1,24 +1,31 @@
 import React from 'react';
+// import { isPropertySignature } from 'typescript';
 import './App.scss';
+import { Clock } from './components/clock';
 
-const App: React.FC = () => {
-  window.setInterval(() => {
-    const date = new Date();
+class App extends React.Component  {
+  state = {
+    isClockVisible: true,
+  }
 
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
+  render() {
+    return(
+      <div className="App">
+        <h1>React clock</h1>
+        <p>
+          Current time:
+          {' '}
+          {this.state.isClockVisible&&<Clock />}
```

Comment:
також використай деструкторизацію  
```suggestion
          {this.state.isClockVisible && (
            <Clock />
          }
```